lib/board.rb

```diff
@@ -0,0 +1,15 @@
+class Board
+  attr_accessor :spaces
```

Comment:
I wonder if this needs to be an accessor?  Would a reader be sufficient?

Reply:
That makes sense, i'll change it!